src/pmt.c

```diff
@@ -716,9 +727,9 @@ void update_cw(SPMT *pmt) {
         }
 
     LOGL(old_cw == cw ? DEFAULT_LOG : 1,
-         "found CW: %d %sfor %s PMT %d, old cw %d, packets %d, parity %d, pid "
+         "found CW: %d %s for %s PMT %d, old cw %d, packets %d, parity %d, pid "
          "%d: %s",
-         cw ? cw->id : -1, validated ? "[validated] " : "", pmt->name, pmt->id,
+         cw ? cw->id : -1, validated ? "[validated]" : "[pending]", pmt->name, pmt->id,
```

Comment:
Why pending? This will be the CW used for decrypting

Reply:
I'm guessing he means it's pending validation?

Reply:
Hi @catalinii ,

> Why pending?

Because this key will be pending until it be validated.

> This will be the CW used for decrypting

Sure? From what I've checked, if a key is not validated (aka "pending"), then it's because one of this:
1. The key is reported but not already checked (because missing PUSI packets).
2. The key is checked with a PUSI packet and the check failed.

Then what the current code does is to mainatin this key in the candidate list of keys. That until in the future a new one arrives or some change is done and the PUSI test will be passed. So a key can only be "validated" or "pending". Why not then print the two status? Printing the key without the "pending" label could induce to think that the key is correct, and in fact it's a key pending to validate it (or pending to be overwrited with another one because is not a valid key). 

Therefore, please accept this change to make more easier to debug key troubles.

Reply:
> I'm guessing he means it's pending validation?

So, you prefer to use the word "unvalidated"? It's that?

Reply:
I think it's fine like this.

Reply:
Changed to "not validated". So comment resolved. 😉